frontend/src/modules/history/components/Translation.css

```diff
@@ -6,7 +6,11 @@
     display: flex;
 }
 
-.history .translation:hover {
+.history .translation.cannot-copy {
+    pointer-events: none;
+}
+
+.history .translation.can-copy:hover {
```

Comment:
This should not be limited to `can-copy` only. Please remove the `.can-only` bit, and since I believe that class is never used, please don't set it in the code.

Reply:
If I am not setting that to `can-copy` then on hovering buttons. Translation background changes

Reply:
Unless I'm misunderstanding, yes, that is the behavior I expect. The background of a translation should become brighter when you are hovering anywhere over it.